### PR TITLE
23 create placeholder profile edit calls

### PR DESCRIPTION
**Pull request by soup-scott**
- Created directory api/updateProfile within src/app
- Added file route.ts with skeleton api handlers

### CODE REVIEW

**Review by SamShrubo — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/BackEnd-feature-template.md
Comment (CesMag) [inline .github/ISSUE_TEMPLATE/BackEnd-feature-template.md:1]: I dont believe you should change our issue template, since we use that to create tickets.

**Review by CesMag — CHANGES_REQUESTED** [review]
Once those changes are removed I can approve.

File: .github/ISSUE_TEMPLATE/BackEnd-feature-template.md
Comment (SamShrubo) [reply .github/ISSUE_TEMPLATE/BackEnd-feature-template.md:1]: it is the same template as was in main so I believe it was originally cloned from there, dev doesn't currently have the issue templates that main does

**Review by SamShrubo — DISMISSED** [review]

**Review by CesMag — APPROVED** [review]
LGTM

**Review by CesMag — APPROVED** [review]

**Review by SamShrubo — APPROVED** [review]

**Review by SamShrubo — DISMISSED** [review]

**Review by CesMag — APPROVED** [review]

**Review by CesMag — DISMISSED** [review]

**Pull request merged**